nipype/scripts/utils.py

```diff
@@ -68,13 +68,35 @@ def add_args_options(arg_parser, interface):
             args["default"] = getattr(inputs, name)
             args["action"] = 'store_true'
 
+        # current support is for simple trait types
+        if not spec.inner_traits:
+            trait_type = type(spec.trait_type.default_value)
+            if trait_type in (str, int, float):
```

Comment:
For compatibility, we should add an `from builtins import str, bytes`. Also, add `bytes` to this check.

Finally, change the type if `trait_type` is `bytes` to be `str`